acai-ios/BowlOrderDetailViewController.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  BowlOrderDetailViewController.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+class BowlOrderDetailViewController: UIViewController {
+    
+    // MARK: View vars
+    
+    var collectionView: UICollectionView!
+    var collectionViewListAdapter: ListAdapter!
+    
+    // MARK: Data
+    
+    var bowlItem: BowlItem!
+    var baseOptions: [CustomizationOption]!
+    var toppingOptions: [CustomizationOption]!
+    
+    // MARK: View Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+        
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        collectionView.backgroundColor = .white
+        collectionView.showsVerticalScrollIndicator = false
+        view.addSubview(collectionView)
+        collectionView.snp.makeConstraints { (make) in
+            make.edges.equalToSuperview()
+        }
+        
+        collectionViewListAdapter = ListAdapter(updater: ListAdapterUpdater(), viewController: self, workingRangeSize: 0)
+        collectionViewListAdapter.collectionView = collectionView
+        collectionViewListAdapter.dataSource = self
+    }
+}
+
+extension BowlOrderDetailViewController: ListAdapterDataSource {
+    func objects(for listAdapter: ListAdapter) -> [ListDiffable] {
+    
+        //hard code
+        
+        bowlItem = Acai.testBowl
+        
+        toppingOptions = bowlItem.toppingOptions
+        baseOptions = bowlItem.baseOptions
+        var list: [ListDiffable] = []
+        list.append("Choose your base" as ListDiffable)
+        list.append(Acai.bowlOrderBaseOptionsIdentifier as ListDiffable)
+        list.append("Choose your toppings" as ListDiffable)
+        list.append(Acai.bowlOrderToppingOptionsIdentifier as ListDiffable)
```

Comment:
spoke offline but we shouldn't use strings as `ListDiffable` models. For example, your `"Choose your base"`, you should create a `ChooseBaseModel`.

i'm thinking you should prob return something like this: 
`[ChooseBaseModel, BaseOptionsModel, ChooseToppingsModel, ToppingOptionsModel]`